app/controllers/DirectDebitController.scala

```diff
@@ -58,7 +60,7 @@ class DirectDebitController @Inject()(val messagesApi: MessagesApi,
               Redirect(url)
             }
 
-          case Left(_) => InternalServerError(directDebitError())
-        }
+          case Left(_) => InternalServerError(standardError(appConfig, "Direct Debit Error", "Error", ""))
```

Comment:
This should be using the messages file as it will not translate to Welsh when eventually do this. Also the error message is incorrect. It should be Sorry, there is a problem with the service. Try again later. The key for this is `vatDetails.error`